db/migrate/20160621151100_polymorphic_mentions.rb

```diff
@@ -0,0 +1,12 @@
+class PolymorphicMentions < ActiveRecord::Migration
+  def change
+    remove_index :mentions, %i(post_id)
```

Comment:
This doesn't work for rollback: `remove_index is only reversible if given a :column option.`